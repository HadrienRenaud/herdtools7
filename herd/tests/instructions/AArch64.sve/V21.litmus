AArch64 V21
{}
P0                     ;
MOV Z0.S,#-1           ;
MOV Z1.S,#2            ;
PTRUE P0.S,VL3         ;
NEG Z1.S,P0/M,Z0.S     ;

forall 0:V1.4S={1,1,1,2}
