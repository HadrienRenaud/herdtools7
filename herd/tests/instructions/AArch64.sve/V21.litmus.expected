Test V21 Required
States 1
0:V1.4S={1,1,1,2};
Ok
Witnesses
Positive: 1 Negative: 0
Flag Scalable-Vector-Extensions-is-work-in-progress
Condition forall (0:V1.4S={1,1,1,2})
Observation V21 Always 1 0
Hash=b2c63565af19e50740733f57fc3bf64a

